smoke-tests/apps/HttpServerQueryString/src/smokeTest/resources/logback-test.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure this file is needed to display the Logback logs on the console. From what I have seen in the Logback code, if no appender is defined then the ConsoleAppender should be added by default.

Reply:
I think it's used to change the root level to warn, but not sure (it's included in all of the smoke tests). We could try it out separate from this PR and remove it from all smoke tests if it's not needed.

Reply:
I _think_ this file is needed for setting the default root level below